internal/policies/dconf/dconf_test.go

```diff
@@ -57,8 +55,8 @@ func TestApplyPolicy(t *testing.T) {
 		"Update machine disabled key with value": {entries: []entry.Entry{
 			{Key: "com/ubuntu/category/key-s", Value: "'onekey-s-othervalue'", Meta: "s"}},
 			isComputer: true, existingDconfDir: "machine-with-disabled-value"},
-		"Machine empty state": {entries: []entry.Entry{}, isComputer: true, existingDconfDir: "-"},
 
+		"No policy still generates a valid db": {entries: nil},
```

Comment:
Maybe add a comment now here to explain why we need it, to ensure if we go back that route, we don’t recreate the same issue :)